packages/sharedServer/src/services/feedItems.server.ts

```
@@ -219,6 +214,41 @@
+    const {feedItemId, accountId, url} = feedItem;
+    const isSafeUrlResult = SharedFeedItemHelpers.validateUrl(url);
+    if (!isSafeUrlResult.success) {
+      return prefixErrorResult(isSafeUrlResult, 'Error validating feed item URL');
+    }
+
+    // Fetch the transcript via youtube-transcript
+    const fetchTranscriptResult = await fetchYouTubeTranscript(url);
+    if (!fetchTranscriptResult.success) {
+      return prefixErrorResult(fetchTranscriptResult, 'Error fetching YouTube transcript');
+    }
+    console.log('fetchTranscriptResult', fetchTranscriptResult.value);
```

ESLint

[error] Unexpected console statement.

(no-console)